integration_tests/__tests__/__snapshots__/custom-reporters-test.js.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Custom Reporters Integration IncompleteReporter for flexibility 1`] = `
+"onRunComplete is called
+Passed Tests: 1
+Failed Tests: 0
+Total Tests: 1
+"
+`;
+
+exports[`Custom Reporters Integration TestReporter with all tests failing 1`] = `
+Object {
+  "onRunComplete": Object {
+    "called": true,
+    "numFailedTests": 1,
+    "numPassedTests": 0,
+    "numTotalTests": 1,
+  },
+  "onRunStart": Object {
+    "called": true,
+    "options": "object",
+  },
+  "onTestResult": Object {
+    "called": true,
+    "times": 1,
+  },
+  "onTestStart": Object {
+    "called": true,
+    "path": false,
+  },
+  "options": Object {
+    "christoph": "pojer",
+    "dmitrii": "abramov",
+    "hello": "world",
+  },
+}
+`;
+
+exports[`Custom Reporters Integration TestReporter with all tests passing 1`] = `
+Object {
+  "onRunComplete": Object {
+    "called": true,
+    "numFailedTests": 0,
+    "numPassedTests": 1,
+    "numTotalTests": 1,
+  },
+  "onRunStart": Object {
+    "called": true,
+    "options": "object",
+  },
+  "onTestResult": Object {
+    "called": true,
+    "times": 1,
+  },
+  "onTestStart": Object {
+    "called": true,
+    "path": false,
+  },
+  "options": Object {
+    "christoph": "pojer",
+    "dmitrii": "abramov",
+    "hello": "world",
+  },
+}
+`;
+
+exports[`Custom Reporters Integration invalid format for adding reporters 1`] = `
+"●  Reporter Validation Error:
+
+Unexpected value for Path at index 0 of reporter at index 0
+ Expected:
+   string
+ Got:
+   number
+ Reporters config:
+   [      3243242    ]
```

Comment:
This whitespace is really ugly. Why can we not just print this using prettyFormat like we always do?